using-mobile-extensions/adobe-edge-identity/README.md

```diff
@@ -0,0 +1,258 @@
+# Adobe Experience Platform Edge Identity
+The Adobe Experience Platform Edge Identity mobile extension enables identity management from your mobile app when using the [Adobe Experience Platform](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/mobile-core) and the [Edge Network extension](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/adobe-edge).
+
+## Configure the Adobe Experience Platform Edge Identity extension in Experience Platform Launch
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Edge Identity** extension, and click **Install**.
+3. There are no configuration settings for **Edge Identity**.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+## Add the AEP Edge Identity extension to your app
+
+### Download and import the Edge Identity extension
+{% hint style="info" %}
+The following instructions are for configuring an application with Edge Identity. If an application will include both Identity and Edge Identity extensions, follow the instructions [below](#download-and-import-the-identity-and-edge-identity-extensions).
```

Comment:
... are for configuring an application using Adobe Experience Platform Edge mobile extensions. If an application includes both Edge Network and Adobe Solution extensions, both the Edge Identity and Identity extensions are required, find more details in the Frequently Asked Questions page <link>.